website/content/docs/lambda/invocation.mdx

```diff
@@ -19,8 +19,23 @@ You can invoke Lambda functions from the Consul service mesh through terminating
 We recommend invoking Lambda functions through terminating gateways. This method supports cross-datacenter communication, transparent
 proxies, intentions, and all other Consul service mesh features.
 
-The terminating gateway must have [the appropriate IAM permissions](/docs/lambda/requirements#allow-envoy-to-invoke-the-lambda)
-to invoke the function.
+The terminating gateway must:
+* Have [the appropriate IAM permissions](/docs/lambda/registration#configure-iam-permissions-for-envoy) to invoke the Lambda function.
+* Have the Lambda function's Consul service linked to it. This is done by:
+  1. Ensuring the Lambda service is linked to the terminating gateway's configuration:
+      <CodeBlockConfig filename="terminating-gateway.hcl">
+
+      ```hcl
+      Kind = "terminating-gateway"
+      Name = "terminating-gateway"
+      Services = [
+        {
+          Name = "<LAMBDA_SERVICE_NAME>"
+        }
+      ]
+      ```
+      </CodeBlockConfig>
```

Comment:
Can you add Kubernetes YAML and JSON examples similar to https://www.consul.io/docs/connect/config-entries/terminating-gateway#access-an-external-service